### DISCUSSION TITLE
Evaluating model like in the first tutorial

### DISCUSSION THREAD

**Discussion by HugoCroizer** [BayraktarLab/cell2location#260] [Q&A]
### Problem
Hi,
I would like to know if it is still possible to have the plot "evaluate_stability.png" like in the old tutorial where we could compare consistency of estimated cell abundances between two independent training restarts ?

Thank you

**Comment by vitkl** [comment]
Hi @Dantferno 

we generally found very high consistency so stopped doing this. You can do it yourself by doing the training twice and computing correlations between cell abundance estimates using `scipy.spatial.distance.cdist(x, y, metric='correlation')`.

**Comment by HugoCroizer** [comment]
Hi @vitkl 

Thank you for your answer, I found this plot to be informative to see if there is a problem distinguishing two populations in the spatial mapping, and see if the resolution on the single cell should be changed. 

What are you thought on that ? We are for example struggling to distinguish different T lymphocytes clusters.  We added lymphocytes from other dataset like you did in your paper for the lymph node and saw an improvement.

Another thing is correcting the single cell based on the regression model like you did in Fig sup 17 of the preprint. With the new tutorial I do it with the following line of code : 
```
 sample = mod.sample_posterior()
 adata_snrna_raw_cor.X = np.array(mod.module.model.normalise(sample['post_sample_means'], adata_snrna_raw_cor))
  
```
It seems to work as before, 
Thank you !

**Comment by vitkl** [comment]
>  I found this plot to be informative to see if there is a problem distinguishing two populations in the spatial mapping

We have not looked into this comparison systematically. Could you please give an example (with plots) of how you used this?

**Comment by vitkl** [comment]
I am assuming that this is resolved.
